core/sql/optimizer/RelExpr.cpp

```diff
@@ -9515,7 +9515,8 @@ FileScan::FileScan(const CorrName& tableName,
      estRowsAccessed_ (0),
      mdamFlag_(UNDECIDED),
      skipRowsToPreventHalloween_(FALSE),
-     doUseSearchKey_(TRUE)
+     doUseSearchKey_(TRUE),
+     computedNumOfActivePartiions_(-1)
```

Comment:
Partitions is misspelled 8 times in this file, might make sense to fix the spelling for all of those.